tools/builder/frontend-css.js

```diff
@@ -103,7 +103,7 @@ gulp.task( 'frontendcss', [ 'frontendcss:separate' ], function() {
 		} );
 } );
 
-gulp.task( 'frontendcss:separate', function() {
+gulp.task( 'frontendcss:separate', gulp.series( 'frontendcss', function() {
```

Comment:
Oh BTW how come we're flipping dependencies here? (Previously, `frontendcss` depended on `frontendcss:separate`, now it's the other way round.)